src/components/Navbar/NavBar.tsx

```diff
@@ -296,10 +318,65 @@ const ReactNavLink = styled(Link)`
     color: black;
   }
   :hover {
-    color: #9e9e9e;
-  }
+    color: #9e9e9e;}
+  position:relative;
 `;
 
+
 const Close = styled(CloseIcon)`
   cursor: pointer;
+`;
+
+
+const Dropdown = styled.div`
+  display:flex;
+  z-index: 11;
+  display: block;
+  position: absolute;
+  left: 145px;
+  top: 19px;
+  width: 213px;
+  height: 183px;  
+  margin:0px;
+  color:black;
+  a:hover {
+    color: #9e9e9e;
+  }
+  background-color:#FFFFFF;
+`;
+
+const DropdownMobile = styled.div`
+  width:375px;
+  height: 163px;  
+  position:relative;
+  top: -8px;
```

Comment:
In general, negative pxs are a code smell. We try to avoid them unless on a tight deadline